### ISSUE TITLE
[HW] Gabriel Oliveira Galvão do Vale

### ISSUE THREAD

**Issue by gogvale** [datastaxdevs/workshop-introduction-to-nosql#553]
**Name:** Gabriel Oliveira Galvão do Vale
**Email:** gabriel.oliveira@axented.com || gogvale2011@gmail.com
**Linkedin Profile:** https://linkedin.com/in/gogvale

Attach the homework screenshots below:
-----------------------------------------
![Screenshot from 2021-08-21 12-23-42](https://user-images.githubusercontent.com/23729423/130330105-2722ac0f-af07-4217-b223-2a6e0388a2b1.png)
![Screenshot from 2021-08-21 12-23-50](https://user-images.githubusercontent.com/23729423/130330106-077b2399-172d-4a74-9b62-aa180b3a2907.png)

**Comment by gogvale** [comment]
![image](https://user-images.githubusercontent.com/23729423/130330210-6018fb32-7353-4fa4-9f21-a342c8fe143e.png)

**Comment by RyanWelford** [comment]
Thanks Gabriel, were you able to complete the tabular and document db portions of the homework? (Parts 2 & 3)

**Comment by gogvale** [comment]
> Thanks Gabriel, were you able to complete the tabular and document db portions of the homework? (Parts 2 & 3)

Yes, I followed the github guide from start to finish (the only thing I didn't do was part V)
Do you need a screenshot as well?

**Comment by gogvale** [comment]
@RyanWelford do you need anything from me to complete the review?
